protocols/syslog/syslog_net.c

```diff
@@ -39,25 +39,28 @@ syslog_net_init(void)
 
   syslog_conn = uip_udp_new(&ip, HTONS(SYSLOG_PORT), syslog_net_main);
 
-  if(! syslog_conn) {
+  if (!syslog_conn)
+  {
+#ifndef DEBUG_USE_SYSLOG
     debug_printf("syslog: couldn't create connection\n");
+#endif
     return;
   }
 
   uip_udp_bind(syslog_conn, HTONS(SYSLOG_PORT));
 
-  syslog_send_P(PSTR("booting ethersex (" VERSION_STRING_LONG ")\n"));
+  syslog_send_P(PSTR("syslog: booting ethersex (" VERSION_STRING_LONG ")"));
```

Comment:
Then i would remove the whole line. Bootup message is already generated by debug. Without the "syslog:" prefix this message lacks the syslogtag and and "booting" gets interpreted as hostname.